types/fundamental-react/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for fundamental-react 0.8
+// Type definitions for fundamental-react 0.9
```

Comment:
There are so many other type changes that need to go into `0.9` and `0.9` is still a `-rc` version so I think we should hold off on creating types for it

Reply:
thats right. so there have to be made some more changes. but currently i don't have the time :-/

Reply:
Does `backAction` exist in 0.8?
- Yes: Leave the version 0.8 and merge.
- No: With regard to the current state of 0.9-rc, are these typings _inaccurate_ or just _incomplete_?
  - Inaccurate: Consider holding off on this PR until corrections can be made.
  - Incomplete: Do we expect these typings to _become_ inaccurate due to breaking changes in the RC before 0.9 is stable?
    - Yes: Consider holding off on this PR until the API is more stable.
    - No: We can merge this as-is and make 0.9 additions later.